core/src/main/java/it/pagopa/selfcare/mscore/core/UserEventServiceImpl.java

```diff
@@ -114,25 +140,36 @@ private UserToNotify toUserToNotify(String userId, String institutionId, User us
         userToNotify.setProductRole(onboardedProduct.getProductRole());
         return userToNotify;
     }
-    public void sendOperatorUserNotification(RelationshipInfo relationshipInfo) {
+    private UserToNotify updateUserToNotify(String userId, OnboardedProduct onboardedProduct){
```

Comment:
naming: probably something like `addProductToUser` would have been clearer ("update" is too generic)